apply.html

```diff
@@ -0,0 +1,116 @@
+                        <h3>Please choose their temperament below</h3>
+                    </label>
+                    <p>
+                    <select name="temperament" id="temperament">
+                        <option value="" hidden disabled selected>select</option>
+                        <option value="calm">calm</option>
+                        <option value="rowdy">rowdy</option>
+                        <option value="hissing">hissing</option>
+                        <option value="screaming">screaming</option>
+                    </select>
+                    </p>
+                    </section>
+                    </p>
+
+                    <p>
+                        <h3>Select your dog's color(s)</h3>
+                    </p>
+                    <section>
+                    <div>
+                        <input type="checkbox" name="colors" value="brown" id="brown">
+                        <label for="brown">brown</label>
+                    </div>
+
+                    <div>
+                        <input type="checkbox" name="colors" value="green" id="green">
+                        <label for="green">green</label>
+                    </div>
+                    
+                    <div>
+                        <input type="checkbox" name="colors" value="white" id="white">
+                        <label for="white">white</label>
+                    </div>
+
+                    <div>
+                        <input type="checkbox" name="colors" value="gray" id="gray">
+                        <label for="gray">gray</label>
+                    </div>
+
+                    <div>
+                        <input type="checkbox" name="colors" value="black" id="black">
+                        <label for="black">black</label>
+                    </div>
+                    </section>
+
+                    <h3>
+                        Are they a good dog?
+                    </h3>
+                    <section>
+                        <input required type="radio" name="good-dog" value="yes" id="yes">
+                        <label for="yes">yes</label>
+
+                        <input required type="radio" name="good-dog" value="yes-caps" id="yes-caps">
+                        <label for="yes-caps">YES</label>
+                    </section>
+                    
+                    <h3>
+                        What is their favorite activity?
+                    </h3>
+                    <section>
+                        <p>
+                            <label for="fave-activity"></label>
+                            <textarea name="fave-activity" id="fave-activity"></textarea>
+                        </p>
+                    </section>
+
+                </fieldset>
+            </section>
+
+            <h2>
+                Click below to submit
+            </h2>
+            <button id="button">Submit</button>
+        </form>
+    </main>
+    <script src="./src/apply.js"></script>
```

Comment:
type="module" needs to be tagged to this script tag

Reply:
YOU'RE THE BEST THANK YOU